content/en/learning/2025-02-07-deepseek-exploring-the-techniques.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "DeepSeek - Exploring the Techniques Behind the Hype"
+slug: "deepseek-exploring-the-techniques"
+draft: false
+event_date: "2025-02-07"
+image: "img/resources/webinars/2025-02-07-deepseek-exploring-the-techniques.png"
+name: "DeepSeek - Exploring the Techniques Behind the Hype"
+description: "Watch us break down some of the key engineering techniques utilized in DeepSeek R1, as well as the potential financial and strategic implications for enterprises."
+events: ['Webinar']
+registration_link:
+call_to_action:
+video_link: https://www.youtube.com/embed/AvvNgvZ3VsU?si=ML4lkLUnbEmO4lTR
+audio_link:
+categories: ['Video']
+presenters: ['Rebecca Bilbro', 'Edwin Schmierer']
```

Comment:
In a follow on story, I will update how we handle users without a bio page. Then I'll add Steve and Ilya as presenters.